pennylane/wires.py

```diff
@@ -283,20 +251,19 @@ def subset(self, indices, periodic_boundary=False):
 
         """
 
+        Len = len(self)
```

Comment:
Syntax highlighting assumes this is a class, maybe better to use all lowercase for the naming (but not `len` to avoid redefining the function). `len_`, or `len_wires`, or `num_wires`.

Reply:
Yes, you are totally right, thanks! I changed it to `len_`